broken imports of signal from nix

https://github.com/sourcefrog/cargo-mutants/actions/runs/8225865418/job/22491508244#step:2:300

```
error[E0432]: unresolved imports `nix::sys::signal::killpg`, `nix::sys::signal::Signal`
   --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-mutants-24.2.1/src/process.rs:163:28
    |
163 |     use nix::sys::signal::{killpg, Signal};
    |                            ^^^^^^  ^^^^^^ no `Signal` in `sys::signal`
    |                            |
    |                            no `killpg` in `sys::signal`
    |
note: found an item that was configured out
   --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.27.1/src/sys/signal.rs:995:8
    |
995 | pub fn killpg<T: Into<Option<Signal>>>(pgrp: Pid, signal: T) -> Result<()> {
    |        ^^^^^^
    = note: the item is gated behind the `signal` feature
    = help: consider importing this type alias instead:
            ctrlc::Signal

error[E0433]: failed to resolve: could not find `Pid` in `unistd`
   --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-mutants-24.2.1/src/process.rs:165:28
    |
165 |     let pid = nix::unistd::Pid::from_raw(child.pid().expect("child has a pid").try_into().unwrap());
    |                            ^^^ could not find `Pid` in `unistd`

```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -88,7 +88,7 @@ version = "2"
 features = ["full", "extra-traits", "visit"]
 
 [target.'cfg(unix)'.dependencies]
-nix = { version="0.28", features = ["signal"] }
+nix = { version="0.28", features = ["process", "signal"] }
 
 [dev-dependencies]
 assert_cmd = "2.0"

